js/constants.js

```diff
@@ -1,10 +1,13 @@
 /*!
- * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2022-2023 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
 module.exports = {
-  CONTEXT_FILENAME: 'data-integrity-v1.jsonld',
-  CONTEXT_URL: 'https://w3id.org/security/data-integrity/v1',
+  CONTEXT_FILENAME_V1: 'data-integrity-v1.jsonld',
+  CONTEXT_FILENAME_V2: 'data-integrity-v2.jsonld',
+  DATA_INTEGRITY_CONTEXT_V1_URL: 'https://w3id.org/security/data-integrity/v1',
+  DATA_INTEGRITY_CONTEXT_V2_URL: 'https://w3id.org/security/data-integrity/v2',
+  CONTEXT_URL: 'https://w3id.org/security/data-integrity/v2',
   CBORLD_VALUE: 0x30
```

Comment:
Does the `CBORLD_VALUE` need to be updated?  Do we need to. make a new value for context v2 in cborld or do we update the URL for 0x30 there? https://github.com/digitalbazaar/cborld/blob/b6fde00c65d5b2adf5fd1bd2849d353a2320f2dc/lib/codecs/registeredContexts.js#L28

Reply:
We need a new `CBORLD_VALUE` for a different context, yes. @davidlehn, what do you say about this and about having more than one context in each of these packages?

Reply:
Updated `CBORLD_VALUE` [12f5357](https://github.com/digitalbazaar/data-integrity-context/pull/4/commits/12f5357c0cf395c4bb5ea66be66f1c8913adf9d5). Related PR - https://github.com/digitalbazaar/cborld/pull/68

Reply:
@dlongley I had a plan long ago for handling this situation, but it was never put in place.  I think multiple contexts in a package is the way to go, but someone seemed to think not doing that was ok: 
https://github.com/digitalbazaar/context-template/pull/4#discussion_r614333263.

I'm not sure what the short term solution is other than export more vars, document it, it will be usable, and we'll fix it later.